LayoutKitTests/TextViewLayoutTests.swift

```diff
@@ -0,0 +1,172 @@
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testLineFragmentPadding() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let text = Text.unattributed(textString)
+        let lineFragmentPadding: CGFloat = 30.0
+
+        let textView = UITextView(
+            text: text,
+            font: UIFont.systemFont(ofSize: 12),
+            lineFragmentPadding: lineFragmentPadding)
+
+        let layout = TextViewLayout(
+            text: text,
+            font: UIFont.systemFont(ofSize: 12),
+            lineFragmentPadding: lineFragmentPadding)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testTextContainerInset() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let text = Text.unattributed(textString)
+        let textContainerInset = UIEdgeInsets(top: 2, left: 3, bottom: 4, right: 5)
+
+        let textView = UITextView(
+            text: text,
+            font: UIFont.systemFont(ofSize: 12),
+            textContainerInset: textContainerInset)
+
+        let layout = TextViewLayout(
+            text: text,
+            font: UIFont.systemFont(ofSize: 12),
+            textContainerInset: textContainerInset)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testInSpecificViewSize() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let text = Text.unattributed(textString)
+
+        let layout = TextViewLayout(text: text)
+        let layoutView = layout.arrangement(origin: CGPoint.zero, width: 20, height: 20)
+
+        XCTAssertTrue(layoutView.frame.size.width <= 20, "Width should be less than the max width")
+        XCTAssertTrue(layoutView.frame.size.height <= 20, "Width should be less than the max height")
+    }
+
+}
+
+// MARK: - private helper extension
+
+private extension UITextView {
+
+    convenience init(text: Text,
+                     font: UIFont? = nil,
+                     lineFragmentPadding: CGFloat = 0,
+                     textContainerInset: UIEdgeInsets = UIEdgeInsets.zero,
+                     contentInset: UIEdgeInsets = UIEdgeInsets.zero,
+                     layoutMargins: UIEdgeInsets = UIEdgeInsets.zero) {
+        self.init()
+
+        switch text {
+        case .unattributed(let unattributedText):
+            self.text = unattributedText
+            if font != nil {
+                self.font = font
+            }
+        case .attributed(let attributedText):
+            // If font is using default, applied default font to attributedString
+            self.attributedText = font != nil
+                ? attributedText.with(defaultFont: font)
+                : attributedText
```

Comment:
Note: can do `self.attributedText = font.map { attributedText.with(defaultFont: $0) }  ?? attributedText`
Up to you which style you think is easier to read.